pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -287,7 +287,44 @@ func describeProjectAndServer(project *projectapi.Project, server string) string
 	}
 }
 
-func describeDeploymentInServiceGroup(deploy graphview.DeploymentConfigPipeline) []string {
+func describeImageStreamTagRuntimeStatus(imageSpec string, c client.Interface, namespace string) string {
```

Comment:
This is a dramatic change from the intent of the rest of this flow.  The entire idea of this command is to build a graph that describes the dependencies and references throughout the system and then analyze that graph to find holes.  Such a representation is fundamentally correct, independently testable, and can very dumped and verified in stages to ensure correctness.  It's also exposable in standard formats and consumeable to other components if we every made an endpoint.

A one-off like this that does a separate client call instead of properly building a complete (enough) graph to begin with is out of keeping with that intent.  This is even more unusual, since I'm pretty sure that `namespace` here is just the namespace that we started with.